src/main/java/org/snomed/aag/rest/AcceptanceController.java

```diff
@@ -23,16 +25,11 @@
     private final BranchSecurityService securityService;
     private final ProjectAcceptanceCriteriaService projectAcceptanceCriteriaService;
     private final AcceptanceService acceptanceService;
-	private final CriteriaItemService criteriaItemService;
-
-    public AcceptanceController(BranchSecurityService securityService,
-			ProjectAcceptanceCriteriaService projectAcceptanceCriteriaService,
-			AcceptanceService acceptanceService, CriteriaItemService criteriaItemService) {
 
+	public AcceptanceController(BranchSecurityService securityService, ProjectAcceptanceCriteriaService projectAcceptanceCriteriaService, AcceptanceService acceptanceService) {
         this.securityService = securityService;
         this.projectAcceptanceCriteriaService = projectAcceptanceCriteriaService;
         this.acceptanceService = acceptanceService;
-		this.criteriaItemService = criteriaItemService;
```

Comment:
The CriteriaItemService dependency was removed (as it was used previously). The constructor changed has resulted in several changes throughout the tests.